nodes/SetComp.py

```diff
@@ -1 +1,30 @@
+"""
+SetComp astroid node
+
+This node represents set comprehension expressions in Python, which take the
+form {element for increment in iterable}. This node also makes use of the
+Comprehension node.
+
+Attributes:
+    - elt         (Expr)
+        - The part to be evaluated to make every item in the set. This can
+          be any expression node, such as a BinOp or Str.
+    - generators  (List[])
```

Comment:
Oops, I just got distracted before finishing that one, thanks for pointing it out.